### PR TITLE
PIP 14 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/lumbering_megasloth.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Lumbering Megasloth
+ManaCost:10 G G
+Types:Creature Sloth Mutant
+PT:8/8
+K:Trample
+K:CARDNAME enters the battlefield tapped.
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each counter among players and permanents.
+SVar:X:SVar$Y/Plus.Z
+SVar:Y:Count$Valid Card.Permanent$CardCounters.ALL
+SVar:Z:PlayerCountPlayers$AllCounters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lumbering_megasloth.txt:10]: needs testing but should work

File: forge-gui/res/cardsfolder/upcoming/plasma_caster.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Plasma Caster
+ManaCost:1 R
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 1 | AddToughness$ 1 | Description$ Equipped creature gets +1/+1.
+T:Mode$ Attacks | ValidCard$ Card.EquippedBy | Execute$ TrigEnergy | TriggerDescription$ Whenever equipped creature attacks, you get {E}{E} (two energy counters).
+SVar:TrigEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2
+A:AB$ FlipACoin | Cost$ PayEnergy<2> | ValidTgts$ Creature.blockingEquipped | WinSubAbility$ ExileIt | LoseSubAbility$ DamageIt | TgtPrompt$ Select target creature blocking equipped creature | SpellDescription$ Choose target creature that's blocking equipped creature. Flip a coin. If you win the flip, exile the chosen creature. Otherwise, Plasma Caster deals 1 damage to it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/plasma_caster.txt:7]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/plasma_caster.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+SVar:ExileIt:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | Defined$ Targeted
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/plasma_caster.txt:8]: wrong origin

File: forge-gui/res/cardsfolder/upcoming/powder_ganger.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Powder Ganger
+ManaCost:2 R
+Types:Creature Human Rogue
+PT:2/2
+K:Squad:2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDestroy | TriggerDescription$When CARDNAME enters the battlefield, destroy up to one target artifact.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/powder_ganger.txt:6]: spacing

File: forge-gui/res/cardsfolder/upcoming/robobrain_war_mind.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Name:Robobrain War Mind
+ManaCost:3 U
+Types:Artifact Creature Robot
+PT:*/5
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ Y | Description$ CARDNAME's power is equal to the number of cards in your hand.
+SVar:Y:Count$InYourHand
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigEnergy | TriggerDescription$ CARDNAME enters the battlefield, you get an amount of {E} (energy counters) equal to the number of artifact creatures you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/robobrain_war_mind.txt:7]: fix description

File: forge-gui/res/cardsfolder/upcoming/synth_infiltrator.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Name:Synth Infiltrator
+ManaCost:3 U U 
+Types:Artifact Creature Synth
+PT:0/0
+K:Improvise
+K:ETBReplacement:Copy:DBCopy:Optional
+SVar:DBCopy:DB$ Clone | Choices$ Creature | AddTypes$ Artifact & Synth | AddKeywords$ Myriad | NonLegendary$ True | SpellDescription$You may have CARDNAME enter the battlefield as a copy of any creature on the battlefield, except it's a Synth artifact creature in addition to its other types.
+DeckHas:Ability$Counters
+Oracle:Improvise (your artifact can help cast this spell. Each artifact you tap after you're done activating mana abilities palys for {1}.)\nYou may have Synth Infiltrator enter the battlefield as a copy of any creature on the battlefield, except it's a Synth artifact creature in addition to its other types.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/synth_infiltrator.txt:9]: fix spacing + oracle

File: forge-gui/res/cardsfolder/upcoming/synth_infiltrator.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Synth Infiltrator
+ManaCost:3 U U 
+Types:Artifact Creature Synth
+PT:0/0
+K:Improvise
+K:ETBReplacement:Copy:DBCopy:Optional
+SVar:DBCopy:DB$ Clone | Choices$ Creature | AddTypes$ Artifact & Synth | AddKeywords$ Myriad | NonLegendary$ True | SpellDescription$You may have CARDNAME enter the battlefield as a copy of any creature on the battlefield, except it's a Synth artifact creature in addition to its other types.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/synth_infiltrator.txt:7]: wrong params

File: forge-gui/res/cardsfolder/upcoming/lumbering_megasloth.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/lumbering_megasloth.txt:10]: Yeah it's working now. (the Radstag as well) 

image

**Review by tool4ever — APPROVED** [review]

**Review by Hanmac — APPROVED** [review]

**Pull request merged**